### ISSUE TITLE
[feature]: Page For Reviewing and Managing Feedbacks.

### ISSUE THREAD

**Issue by sakshi21-03** [garg3133/JagratiWebApp#137]
To create a new page which shows feedbacks.
can you please assign me for GSSOC'21 label.

**Comment by garg3133** [comment]
Sure @sakshi21-03, you can continue working on this. What we exactly want is this:

Create a new page to show all the feedbacks received. The feedbacks should be displayed in form of cards with the name and email of the author at the top, then feedback (truncated to a specific no. of characters) and date at the bottom.

On clicking over the feedback card, a modal should open displaying the complete feedback.

Let me know if you face any problem. Assigning you the issue.

**Comment by sakshi21-03** [comment]
> Sure @sakshi21-03, you can continue working on this. What we exactly want is this:
> 
> Create a new page to show all the feedbacks received. The feedbacks should be displayed in form of cards with the n

> Sure @sakshi21-03, you can continue working on this. What we exactly want is this:
> 
> Create a new page to show all the feedbacks received. The feedbacks should be displayed in form of cards with the name and email of the author at the top, then feedback (truncated to a specific no. of characters) and date at the bottom.
> 
> On clicking over the feedback card, a modal should open displaying the complete feedback.
> 
> Let me know if you face any problem. Assigning you the issue.

**Comment by sakshi21-03** [comment]
@garg3133 , okay i got it, but for this feedback  showing page, first we need to create a page where user will  be able to give feedback. Please let me know about the same.

**Comment by garg3133** [comment]
@sakshi21-03 Yeah! We already have one at `/feedbacks`.

**Comment by garg3133** [comment]
Any progress @sakshi21-03?

**Comment by sakshi21-03** [comment]
> Any progress @sakshi21-03?

Yes @garg3133 , i will report you by the end of the day.

**Comment by garg3133** [comment]
@sakshi21-03 What is the progress on this?

**Comment by garg3133** [comment]
@Haneesha123 Would you like to work on this one?

**Comment by Haneesha123** [comment]
Yes @garg3133. I would like to try.

**Comment by garg3133** [comment]
Assigned the issue to you.

**Comment by garg3133** [comment]
@Haneesha123 Any updates on this?

**Comment by Haneesha123** [comment]
Sorry, I have been busy with academics. So, I couldn't start working on this yet. I will start soon.

**Comment by coding-geek21** [comment]
@garg3133 can I work on this issue?

**Comment by Haneesha123** [comment]
@coding-geek21 I am working on this issue

**Comment by Haneesha123** [comment]
@garg3133 I don't think I will be able to work on this issue. I am un-assigning myself.

**Comment by garg3133** [comment]
No worries @Haneesha123, you can take up some other issue and work on that whenever you get free.

**Comment by coding-geek21** [comment]
> Hello @coding-geek21, you claimed this issue to work on it, but this issue and any referenced pull requests haven't been updated for 10 days. Are you still working on this issue?
> 
> If so, please update this issue by leaving a comment on this issue to let me know that you're still working on it. Otherwise, I'll automatically remove you from this issue in 4 days.
> 
> If you've decided to work on something else, simply unassign yourself from this issue by going to the `Assignees` section and clicking on `unassign me`, so that someone else can claim it and continue from where you left off.
> 
> Thank you for your valuable contributions to JagratiWebApp!

Yes i'm working on this issue , since i was having exams i was not able to complete, i will complete as soon as possible
